$NetBSD$

Remove tests from directory list.
Add target "test" to build test programs.

--- Makefile.orig	2016-09-30 11:33:14.000000000 +0000
+++ Makefile
@@ -16,7 +16,7 @@
 
 include makeinclude
 
-DIRS = $(IMAGEDIRS) $(LIBDECORDIR) src $(CAIRODIR) fluid test documentation
+DIRS = $(IMAGEDIRS) $(LIBDECORDIR) src $(CAIRODIR) fluid documentation
 
 all: makeinclude fltk-config
 	for dir in $(DIRS); do\
@@ -24,6 +24,14 @@ all: makeinclude fltk-config
 		(cd $$dir; $(MAKE) $(MFLAGS)) || exit 1;\
 	done
 
+test: makeinclude
+	dir=test;\
+	echo "=== making $$dir ===";\
+	if test ! -f $$dir/makedepend; then\
+		touch $$dir/makedepend;\
+	fi;\
+	(cd $$dir; $(MAKE) $(MFLAGS)) || break;\
+
 install: makeinclude
 	-mkdir -p $(DESTDIR)$(bindir)
 	$(RM) $(DESTDIR)$(bindir)/fltk-config
